Psr17: prevet usage of deprecated StreamFactory and use StreamFactoryInterface instead (#87)
php-http/cache-plugin: CHANGELOG.md

# Change Log

# Version 2

> ## 2.0.0 - XXXX-XX-XX

Reviewer: ```suggestion
## 2.0.0 - (unreleased)
```

### Changed

> - Drop support for using `StreamFactory`, only PSR-17 `StreamFactoryInterface` is now supported.

Reviewer: ```suggestion
- Drop support of deprecated PHP-HTTP `StreamFactory`, only PSR-17 `StreamFactoryInterface` is now supported.
```
